Dockerfile

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM centos:centos7
+LABEL maintainer="Presto community <https://prestosql.io/community.html>"
+
+ENV JAVA_HOME /usr/lib/jvm/java-11
+RUN \
+    set -xeu && \
+    yum -y update && \
+    yum -y install java-11-openjdk-devel less && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    rm -rf /tmp/* /var/tmp/*
+
+RUN \
+    set -xeu && \
+    mkdir -p /data/presto
+
+ARG PRESTO_VERSION
+ARG PRESTO_LOC=presto-server/target/presto-server-${PRESTO_VERSION}.tar.gz
+ADD ${PRESTO_LOC} /tmp
```

Comment:
Let's use `PATH` instead of `LOC` so that it's not abbreviated

Reply:
Using `*_LOCATION` as `PRESTO_LOC` and `CLIENT_LOC` are not necessarily paths, but can be URLs.